demisto_sdk/commands/unify/generic_module_unifier.py

```diff
@@ -0,0 +1,61 @@
+"""
+Unifies a GenericModule object with it's Dashboards
+"""
+import json
+import os
+from typing import Dict, Optional
+
+
+def find_dashboard_by_id(pack_path: str, dashboard_id: str) -> Optional[Dict]:
+    """
+    Search for dashboard with the given id in the given pack path.
+
+    Args:
+        pack_path: a pack to search in
+        dashboard_id: dashboard id to search for
+
+    Returns: if found - the content of the dashboard, else - None.
+    """
+    for file_name in os.listdir(pack_path):
```

Comment:
no need to look in the entire pack for a dashboard - it will be in the `Dashboards` dir